diff --git a/node_modules/util/util.js b/node_modules/util/util.js
index 6eea657..eaa0226 100644
--- a/node_modules/util/util.js
+++ b/node_modules/util/util.js
@@ -614,14 +614,14 @@ exports.promisify = function promisify(original) {
     throw new TypeError('The "original" argument must be of type Function');
 
   if (kCustomPromisifiedSymbol && original[kCustomPromisifiedSymbol]) {
-    var fn = original[kCustomPromisifiedSymbol];
-    if (typeof fn !== 'function') {
+    var _fn = original[kCustomPromisifiedSymbol];
+    if (typeof _fn !== 'function') {
       throw new TypeError('The "util.promisify.custom" argument must be of type Function');
     }
-    Object.defineProperty(fn, kCustomPromisifiedSymbol, {
-      value: fn, enumerable: false, writable: false, configurable: true
+    Object.defineProperty(_fn, kCustomPromisifiedSymbol, {
+      value: _fn, enumerable: false, writable: false, configurable: true
     });
-    return fn;
+    return _fn;
   }
 
   function fn() {
